scripts/generate_settings_schema.py

```diff
@@ -0,0 +1,13 @@
+import json
+
+from prefect import __development_base_path__
+from prefect.settings import Settings
+
+
+def main():
+    with open(__development_base_path__ / "schemas" / "settings_schema.json", "w") as f:
+        json.dump(Settings.model_json_schema(), f, indent=4)
```

Comment:
jw, how did you settle on `indent=4`?

Reply:
No strong reason, it just matches what I have the formatter in my IDE set to.